HealthcareApp/src/Healthcare.Domain/Constants/DefaultWorkingHours.cs

```diff
@@ -4,9 +4,7 @@ namespace Domain.Constants;
 
 public static class DefaultWorkingHours
 {
-    public static readonly IReadOnlyList<Schedule> Schedules;
-
-    static DefaultWorkingHours()
+    public static IReadOnlyList<Schedule> CreateWorkingHours(string doctorId)
     {
         var workingHours = new List<Schedule>();
```

Comment:
This code can be made more readable

Reply:
thank you for code review. In what way I can achieve more readability here?

Reply:
This is not a critical error, but you can calculate the start of the week once outside the loop.